packages/orchestrator/internal/sandbox/network/slot.go

```diff
@@ -74,6 +75,8 @@ type Slot struct {
 	hostIp   net.IP
 	hostNet  *net.IPNet
 	hostCIDR string
+
+	hyperloopIP string
```

Comment:
Just to make it clearer that Hyperloop IP is only dependent on slot routing and theoretically can be different for each sandbox.